ops/Makefile

```diff
@@ -13,7 +13,7 @@ CURL_TIMEOUT?=20
 
 # Internal target: check if the passed-in wildcard is a known environment name. Hard-coding them because let's be real here.
 .valid-env-%:
-	@case $* in dev|stg|prod|prod-dmz) ;; *) echo "$* is not a valid environment"; exit 1;; esac
+	@case $* in dev|stg|prod|prod-dmz|global) ;; *) echo "$* is not a valid environment"; exit 1;; esac
 
 api.tfvars: /dev/null
```

Comment:
It seems like this isn't used anywhere. Should we remove it?

Reply:
`global` has code written for it that will be available in the next couple of PRs coming out! This was anticipatory. 😄

Reply:
Thanks!